app/models/sandwich.rb

```diff
@@ -6,4 +6,6 @@ class Sandwich < ActiveRecord::Base
   attr_encrypted :oauth_grant_code, key: ENV['ENCRYPTION_KEY']
   attr_encrypted :access_token, key: ENV['ENCRYPTION_KEY']
   attr_encrypted :refresh_token, key: ENV['ENCRYPTION_KEY']
+
+  enum plan: [:test, :pbj, :blt]
```

Comment:
I honestly didn't know about this `enum` method.  Maybe to keep this simple, just validate the value against an array of strings? 

I don't want to assume too much just because I wasn't familiar with this, but this feels like something a Joe/Jane average dev would have to look up.

Reply:
Also, plan names are immutable, so throwing this in an enum kind of makes you think they might be mutable.

Reply:
updated at 674e37d